### DISCUSSION TITLE
[Question][Hw0-2] oUnit not found

### DISCUSSION THREAD

**Discussion by lummsklr** [prosyslab-classroom/cs348-information-security#641] [Q&A]
Name: 김명규

Hello, I install ocaml environment myself with 'DiskuvOCaml', but there is some problem.
Testing with utop is fine, but an error occurs when I enter "dune exec test/sigmaTest.exe" in terminal like this:
------------------------------------------------------------------------------
```
dune exec test/sigmaTest.exe     
File "test/dune", line 1, characters 40-45:
1 | (test (name sigmaTest) (libraries sigma oUnit) (modules sigmaTest))
                                            ^^^^^
Error: Library "oUnit" not found.
-> required by _build/default/test/sigmaTest.exe
```
-----------------------------------------------------------------------------
Even though I enter "opam install oUnit", it does not change.
How can I fix it?
Thank you.

**Comment by KAIST-JongchanPark** [comment]
We recommend to use given docker image, or KCLOUD VM (will be provided soon).
